src/Symfony/Component/Serializer/Debug/TraceableNormalizer.php

```diff
@@ -23,14 +23,13 @@
  * Collects some data about normalization.
  *
  * @author Mathias Arlaud <mathias.arlaud@gmail.com>
- *
- * @internal
```

Comment:
should be in its own PR for 6.4 + replaced by `@final`

Reply:
Done, see #58013